README.md

```diff
@@ -335,3 +335,74 @@ The response contains the same data as when the repair was created:
 
 As above: if the repair was created without work orders, an empty array of
 workOrders should be returned (rather than no key or a null value)
+
+
+## Get Appointments for Work Order
+
+Returns a list of appointments matching the given criteria.
+
+```
+GET /v1/appointments/
+```
+
+### Parameters
+
+- workorderreference (required)
+
+### Response
+A successful response should create the Work order in DRS and get the list of appointments.
+
+```json
+{
+  "metadata": {
+    "resultset": {
+      "count": 5,
+      "offset": 0,
+      "limit": 10
```

Comment:
How will these be specified by the calller? I suggest as query parameters. This is something we missed when specifying the list properties endpoint. See https://github.com/WhiteHouse/api-standards#record-limits - maybe  this should be a separate section, rather than repeated for every "list" endpoint.

Reply:
@dgmstuart FYI pagination is already in our API standards document - https://github.com/LBHackney-IT/api-standards#pagination

Reply:
@leeky will delete this comment